static/css/stylesheet.css

```diff
@@ -159,9 +160,12 @@ h1 {
 }
 
 h2 {
-  padding-bottom: 10px;
+  padding-bottom: 0px;
+  margin-bottom: 20px;
   font-size: 32px;
-  background: url('../images/bg_hr.png') repeat-x bottom;
+  /*background: url('../images/bg_hr.png') repeat-x bottom;*/
```

Comment:
OK to just remove both lines altogether -- we don't check in commented-out code. It's already in the version history, anyway.